SCDoc/lex.scdoc.cpp

```diff
@@ -1,18 +1,24 @@
-#line 2 "lex.scdoc.cpp"
```

Comment:
Are the changes in this file supposed to be a part of this PR? Just checking

Reply:
yes, the `reserved` keyword used here isn't valid in C++17 so i needed to regenerate the file. was the commit message unclear?

Reply:
No no, it's all good. I failed to check commit messages, it's clearly explained. Thanks for that!